webapp/src/components/NewsList.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div>
+    <news-item
+        v-for="newsItem in sortedItems"
+        v-bind:title="newsItem.title"
+        v-bind:votes="newsItem.votes"
```

Comment:
```suggestion
       :item="newsItem"
```
Why not pass the entire object as a property? :thinking: